uber/utils.py

```diff
@@ -14,8 +14,6 @@ def __init__(self, page, *args, **kwargs):
         args = [self.quote(s) for s in args]
         kwargs = {k: self.quote(v) for k, v in kwargs.items()}
         cherrypy.HTTPRedirect.__init__(self, page.format(*args, **kwargs))
```

Comment:
I think removing this is going to be fine, but I'll find out in testing. I'd like to remove it if possible.

I *think* the only downside is that a request coming in via non-SSL will get one additional 302 redirect by the external proxy.  I think that's a good tradeoff here in order to make our code a little more robust in not having to know it's own URL, and that there's only two scenarios where we need this:
1) User managed to reach ubersystem port by not using HTTP (which shouldn't be possible because that should get redirected first by nginx to HTTPS)
2) Internally, we're somehow building an absolute redirect URL to ourselves that includes HTTP (which, I don't think we'd ever do), but even if we did, the proxy server should redirect that request as well.